### PR TITLE
Update readme files with the necessary information to run the project 

### PR DESCRIPTION
**Pull request by devjoaov**
[Linear issue](https://linear.app/bleu-llc/issue/BAL-72/create-readme-and-ensure-code-is-documented)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,34 +1,64 @@
...
+$ git clone --recurse-submodules https://github.com/bleu-llc/balancer-pool-metadata.git
+```
 
 ## Installation
 
 This project requires node v18.14.0. We suggest installing it using [asdf](https://asdf-vm.com/)
 
 ```bash
-git clone https://github.com/asdf-vm/asdf.git ~/.asdf --branch v0.11.1
+$ git clone https://github.com/asdf-vm/asdf.git ~/.asdf --branch v0.11.1
```
Comment (ribeirojose) [inline README.md:44]: 😭

File: packages/contracts/README.md [hunk 2]
```diff
@@ -1,47 +1,42 @@
-# <h1 align="center"> Hardhat x Foundry Template </h1>
+# <h1 align="center"> Balancer Pool Metadata Contract</h1>
 
-**Template repository for getting started quickly with Hardhat and Foundry in one project**
+This package contains the source code for the Balancer Pool Metadata smart-contract, the [`PoolMetadataRegistry`](./src/PoolMetadataRegistry.sol).
```
Comment (ribeirojose) [inline packages/contracts/README.md:3]:
```suggestion
This package contains the source code for the Balancer Pool Metadata Smart Contract, under `src/PoolMetadataRegistry.sol:PoolMetadataRegistry`.
```

File: packages/contracts/README.md [hunk 3]
```diff
@@ -1,47 +1,42 @@
...
 
-![Github Actions](https://github.com/devanonon/hardhat-foundry-template/workflows/test/badge.svg)
+# Getting started
 
-### Getting Started
+## Filling environment variables
```
Comment (ribeirojose) [inline packages/contracts/README.md:7]:
```suggestion
## Environment variables
```

File: packages/contracts/README.md [hunk 4]
```diff
@@ -1,47 +1,42 @@
...
 
-- Use Foundry:
+To be able to test and deploy the smart contract, you need to create a `.env` file in the `contracts` directory. Check the [`.env.example`](./.env.template) for information.
```
Comment (ribeirojose) [inline packages/contracts/README.md:9]:
```suggestion
Make sure to create your own `.env` (`cp .env.template .env`) and add your variables before testing and trying to deploy the contract.
```

File: packages/contracts/README.md [hunk 5]
```diff
@@ -1,47 +1,42 @@
...
-```
+## Install dependencies
 
-- Use Hardhat:
+If you did not clone the repository using the `--recursive-submodules` option, make sure to install the submodules with
 
 ```bash
-npm install
-pnpm hardhat test
+$ git submodule update --init --recursive   # in the balancer-pool-metadata directory
```
Comment (ribeirojose) [inline packages/contracts/README.md:16]: `#question` does it make a difference if you run this outside of those directories?

```suggestion
$ git submodule update --init --recursive
```

File: packages/contracts/README.md [hunk 6]
```diff
@@ -1,47 +1,42 @@
...
 
 ```bash
-pnpm hardhat example
+$ pnpm test
 ```
 
-- Install libraries with Foundry which work with Hardhat.
+## Deploy to local
+
+We are using an anvil to create a local testnet node for deploying and testing smart contracts. Run the anvil server, get a generated private key in the terminal, and change your `PRIVATE_KEY` in the `.env`. Then deploy with the `deploy:local` command.
```
Comment (ribeirojose) [inline packages/contracts/README.md:31]:
```suggestion
We are using [anvil](https://book.getfoundry.sh/anvil/) to create a local testnet node for deploying and testing contracts. Run the anvil server, get a generated private key in the terminal, and change your `PRIVATE_KEY` in the `.env` file. Then deploy with the `deploy:local` command.
```

File: packages/contracts/.env.template [hunk 7]
```diff
@@ -1,7 +1,13 @@
...
+# ETHERSCAN_API_KEY: This API key is used to verify and publish smart contracts on the Etherscan blockchain explorer. To get an API key, follow these steps in the etherscan documentation -> https://docs.etherscan.io/getting-started/viewing-api-usage-statistics
 ETHERSCAN_API_KEY=
-FORGE_RPC_URL=
-FORGE_PRIVATE_KEY=
+
+# PRIVATE_KEY: This is the private key for the Ethereum account that will be used to deploy the smart contract to the Goerli testnet. To obtain a private key, you can follow the steps in the metamask documentation -> https://support.metamask.io/hc/en-us/articles/360015289632-How-to-export-an-account-s-private-key).
 PRIVATE_KEY=
+
+# GOERLI_RPC_URL: This is the RPC endpoint for the Goerli testnet. To obtain the RPC URL, follow these steps: Go to your wallet, click on the Goerli network and look for the RPC URL value.
 GOERLI_RPC_URL=
```
Comment (ribeirojose) [inline packages/contracts/.env.template:13]: `#would` also add what these things are for -- [anvil](https://book.getfoundry.sh/anvil/) for the local testnet,  etherscan for verifying contracts, private key to sign deploys, and RPC to connect to the testnet.

**Review by ribeirojose — APPROVED** [review]
🤝

File: README.md [hunk 8]
```diff
@@ -1,34 +1,64 @@
-# Balancer Pool Metadata
+# <h1 align="center"> Balancer Pool Metadata </h1>
+
+Balancer Pool Metadata is an open source project that enables pool owners to add metadata to their pools on the Balancer platform. This metadata can include information such as pool name, symbol, description, link and more.
+
+By adding metadata to their pools, pool owners can improve the discoverability of their pools on Balancer and make it easier for users to understand the purpose and characteristics of the pool.
+
+The Balancer Pool Metadata project provides an easy-to-use interface for pool owners to add and update their pool metadata, as well as a public subgraph for developers to access and use the metadata.
```
Comment (mendesfabio) [inline README.md:7]:
```suggestion
Balancer Pool Metadata is an open-source project that enables pool owners to add metadata to their pools on the Balancer platform. This metadata can include information such as pool name, symbol, description, link, and much more.

By adding metadata to their pools, pool owners can improve the discoverability of their pools on Balancer and make it easier for users to understand the purpose and characteristics of the pool.

The Balancer Pool Metadata project provides an easy-to-use interface for pool owners to add and update their pool metadata, as well as a public Subgraph for developers to access and use the metadata.
```

File: packages/contracts/README.md [hunk 9]
```diff
@@ -1,47 +1,42 @@
...
-- Use Hardhat's task framework
+## Running tests
 
 ```bash
-pnpm hardhat example
+$ pnpm test
 ```
 
-- Install libraries with Foundry which work with Hardhat.
+## Deploy to local
```
Comment (mendesfabio) [inline packages/contracts/README.md:29]:
```suggestion
## Deploy to local network
```

File: packages/contracts/README.md [hunk 10]
```diff
@@ -1,47 +1,42 @@
...
+We are using an anvil to create a local testnet node for deploying and testing smart contracts. Run the anvil server, get a generated private key in the terminal, and change your `PRIVATE_KEY` in the `.env`. Then deploy with the `deploy:local` command.
 
 ```bash
-forge install rari-capital/solmate # Already in this repo, just an example
+$ pnpm dev:anvil
+$ pnpm deploy:local
 ```
 
-### Notes
+## Deploy to goerli testnet
```
Comment (mendesfabio) [inline packages/contracts/README.md:38]:
```suggestion
## Deploy to Goerli testnet
```

File: packages/contracts/README.md (hunk 5 above)
Comment (devjoaov) [reply packages/contracts/README.md:16]: I tried to run it in the contract directory and it did not work

**Review by ribeirojose — APPROVED** [review]

**Pull request merged**